tests/gdx-tests/src/com/badlogic/gdx/tests/Scene2dTest.java

```diff
@@ -167,6 +169,26 @@ public void run () {
 		window.pack();
 		window.pack();
 		stage.addActor(window);
+		
+		ImageTextButton buttonLeft = new ImageTextButton("HI IM LEFT", skin, "default", true);
+		ImageTextButton buttonRight = new ImageTextButton("HI IM RIGHT", skin, "default", false);		
+		CheckBox checkBoxLeft = new CheckBox("HI IM LEFT", skin, "default", true);
+		CheckBox checkBoxRight = new CheckBox("HI IM RIGHT", skin, "default", false);
+		
+		buttonLeft.setPosition(300, 400);
+		buttonRight.setPosition(300, 370);		
+		checkBoxLeft.setPosition(150, 400);
+		checkBoxRight.setPosition(150, 370);;

```

Comment:
Well, a duplicated ';' is problematic only after a return because it's considered dead code. Stupid compiler!  :)